spec/globus/client_spec.rb

```diff
@@ -83,6 +83,34 @@
     end
   end
 
+  describe ".user_exists?" do
+    let(:fake_instance) { instance_double(described_class) }
+
+    before do
+      allow(described_class).to receive(:instance).and_return(fake_instance)
+      allow(fake_instance).to receive(:user_exists?)
+    end
+
+    it "invokes instance#user_exists?" do
+      described_class.user_exists?
+      expect(fake_instance).to have_received(:user_exists?).once
+    end
+  end
+
+  describe "#exists?" do
```

Comment:
I believe this should still be `#user_exists?` since that's the method defined on the singleton instance.